Packs/DeCYFIR/Integrations/decyfiriocs/decyfiriocs.yml

```diff
@@ -22,93 +22,87 @@ configuration:
   defaultvalue: feedInstanceReputationNotSet
-  type: 18
-  required: false
+  display: Indicator Reputation
+  name: feedReputation
   options:
   - None
   - Good
   - Suspicious
   - Bad
-  additionalinfo: Indicators from this integration instance will be marked with this reputation.
-- display: Source Reliability
-  name: feedReliability
+  required: false
+  type: 18
+- name: feedReliability
+  additionalinfo: Reliability of the source providing the intelligence data.
   defaultvalue: F - Reliability cannot be judged
-  type: 15
-  required: true
+  display: Source Reliability
   options:
   - A - Completely reliable
   - B - Usually reliable
   - C - Fairly reliable
   - D - Not usually reliable
   - E - Unreliable
   - F - Reliability cannot be judged
-  additionalinfo: Reliability of the source providing the intelligence data.
-- display: ""
-  name: feedExpirationPolicy
-  defaultvalue: indicatorType
-  type: 17
+  required: true
+  type: 15
+- name: feedExpirationPolicy
+  defaultvalue: ''
   required: false
-  options:
-  - never
-  - interval
-  - indicatorType
-  - suddenDeath
-- display: ""
-  name: feedExpirationInterval
-  defaultvalue: "20160"
   type: 1
+- defaultvalue: '20160'
+  name: feedExpirationInterval
   required: false
-- display: Feed Fetch Interval
+  type: 1
+- defaultvalue: '240'
+  display: Feed Fetch Interval
   name: feedFetchInterval
-  defaultvalue: "240"
-  type: 19
   required: false
-- display: Bypass exclusion list
-  name: feedBypassExclusionList
+  type: 19
+- additionalinfo: When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.
   defaultvalue: 'true'
-  type: 8
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
   required: false
-  additionalinfo: When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.
-- name: feedTags
+  type: 8
+- additionalinfo: Supports CSV values.
   display: Tags
+  name: feedTags
   required: false
   type: 0
-  additionalinfo: Supports CSV values.
-- name: tlp_color
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed
   display: Traffic Light Protocol Color
+  name: tlp_color
   options:
   - RED
   - AMBER
   - GREEN
   - WHITE
   required: false
   type: 15
-  additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed
+description: DeCYFIR API's provides External Threat Landscape Management insights.
+display: DeCYFIR Indicators & Threat Intelligence Feed
+name: DeCYFIR Feed
 script:
+  commands:
+  - deprecated: false
+    description: Gets indicators from the feed.
+    execution: false
+    name: decyfir-get-indicators
   dockerimage: demisto/python3:3.10.13.84405
```

Comment:
Please change to the latest docker:

dockerimage: demisto/python3:3.11.9.107902